dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -1234,6 +1662,85 @@ internal class RumViewScopeTest {
         assertThat(testedScope.activeActionScope).isNull()
     }
 
+    @Test
+    fun `𝕄 wait for pending 𝕎 handleEvent(StartAction) on active view`(
+        @Forgery type: RumActionType,
+        @StringForgery name: String,
+        @BoolForgery waitForStop: Boolean
+    ) {
+        // Given
+        testedScope.activeActionScope = null
+        testedScope.pendingActionCount = 0
+        fakeEvent = RumRawEvent.StartAction(type, name, waitForStop, emptyMap())
+
+        val result = testedScope.handleEvent(fakeEvent, mockWriter)
```

Comment:
same here, maybe we should send also a `StopView` event to verify this behaviour ?

Reply:
Same here